src/blueapi/worker/task.py

```diff
@@ -32,16 +32,6 @@ def do_task(self, ctx: BlueskyContext) -> None:
         ctx.run_engine(wrapped_plan_generator)
 
 
-# Here for backward compatibility pending
-# https://github.com/DiamondLightSource/blueapi/issues/253
-class RunPlan(Task):
```

Comment:
Is this still required for backwards compatibility? Is it referenced anywhere?

Reply:
to be honest RunEngineWorker shares a name, but might be renamed into TaskWorker

Reply:
renamed